ps_mbo.php

```diff
@@ -139,14 +139,14 @@ class ps_mbo extends Module
     public function __construct()
     {
         $this->name = 'ps_mbo';
-        $this->version = '2.0.2';
+        $this->version = '2.0.3';
         $this->author = 'PrestaShop';
         $this->tab = 'administration';
         $this->module_key = '6cad5414354fbef755c7df4ef1ab74eb';
         $this->need_instance = 0;
         $this->ps_versions_compliancy = [
             'min' => '1.7.5.0',
-            'max' => _PS_VERSION_,
+            'max' => '8.0.0',
```

Comment:
Real question : Is it enough to avoid execution of the module in shops < 8 ?

Reply:
The module will still be compatible with older version, but it will only be to 8.0.0 and not after (can we make a 8.0.x here ?)